tests/integration.rs

```diff
@@ -38,6 +38,33 @@ mod http_tests {
     }
 }
 
+#[cfg(test)]
+mod mongo_tests {
+    use super::runtime::*;
+    use anyhow::Result;
+    use mongo_wasmtime::WasiMongoClient;
+    use wasmtime::Linker;
+
+    const MONGO_RUST_TEST: &str =
+    "tests/modules/rust-mongo/target/wasm32-wasi/release/rust_mongo.wasm";
+    const MONGO_SERVER_ADDR: &str = "mongodb://localhost:27017";
```

Comment:
I think this requires:

- Mongo is configured locally
- Mongo is started for the test.

Have a look at the Redis test controller for an example of how to handle the server for the test.

Reply:
Any idea why the test would still pass? Without the server the connection should fail am I right. I even forced returning an Err from the check_connection function in the test modules yet it still passes :/

Reply:
I think this is #1 ...

Reply:
Ah makes sense. I can test it out through my own test guest module run I suppose. But thanks for pointing it out.

Reply:
See https://github.com/fermyon/wasi-experimental-toolkit/blob/e56ce201cb230381a443ce2f3d8c975776771816/tests/integration.rs#L139-L175